docs/user/user/pings/index.md

```diff
@@ -95,21 +225,91 @@ where:
-| `Date` | e.g. `Mon, 23 Jan 2019 10:10:10 GMT+00:00` | Submission date/time in GMT/UTC+0 offset |
-| `X-Client-Type` | `Glean` | Custom header to support handling of Glean pings in the legacy pipeline |
-| `X-Client-Version` | e.g. `0.40.0` | The Glean SDK version, sent as a custom header to support handling of Glean pings in the legacy pipeline |
-| `X-Debug-ID` | *Optional*, e.g. `test-tag` | Debug header attached to Glean pings when using the [debug tools](../../user/debugging/index.md) |
-| `X-Source-Tags` | *Optional*, e.g. `automation, perf` | A list of tags to associate with the ping, useful for clustering pings at analysis time, for example to tell data generated from CI from other data. |
+
+A pre-defined set of headers is additionally sent along with the submitted ping.
+
+#### `Content-Type`
+
+Describes the data sent to the server. Value is always `application/json; charset=utf-8`.
+
+#### `Date`
+
+Submission date/time in GMT/UTC+0 offset e.g. `Mon, 23 Jan 2019 10:10:10 GMT+00:00`.
+
+#### `User-Agent`
+
+The Glean SDKs do not set this header[^1], so it will contain whatever value was set
+by the underlying uploading mechanism. For example, when sending pings from browsers
+it will contain the characteristic [browser UA string](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/User-Agent#syntax).
+
+This header is parsed by the Glean pipeline and can be queried at analysis time through
+the [`metadata.user_agent.*`](https://docs.telemetry.mozilla.org/datasets/pings.html?highlight=metadata#ping-metadata)
+fields in the ping tables.
+
+#### `X-Telemetry-Agent`
+
+This header is useful for debugging purposes when pings are sent to [the error stream](https://docs.telemetry.mozilla.org/concepts/pipeline/filtering.html?highlight=error%20stream#querying-the-error-stream),
+as it describes the application and the Glean SDK used for sending the ping.
+
+It's looks like `Glean/40.0.0 (Kotlin on Android)`, where `40.0.0` is the Glean SDK version number
+and `Kotlin on Android` is the name of the language used by the SDK that sent the request
+plus the name of the platform it is running on.
+
+_This header is currently only sent by the Glean JavaScript SDK. See [note](#1)._
+
+#### `X-Client-Type`
+
+Custom header to support handling of Glean pings in the legacy pipeline. Values is always `Glean`.
+
+#### `X-Client-Version`
+
+The Glean SDK version e.g. `0.40.0`, sent as a custom header to support handling of Glean pings in the legacy pipeline.
+
+#### `X-Debug-Id` _(optional)_
+
+Debug header attached to Glean pings by using the [debug APIs](../../reference/debug/debugViewTag.md)
+e.g. `test-tag`.
+
+When this header is present, the ping is redirected to the
+[Glean Debug View](http://localhost:3000/user/debugging/index.html#glean-debug-view).
```

Comment:
wrong url.

```suggestion
[Glean Debug View](../../debugging/index.md#glean-debug-view).
```

(double check if this is correct please)

Reply:
```suggestion
[Glean Debug View](../debugging/index.md#glean-debug-view).
```

Actually only one level down.